### ISSUE TITLE
Is "--file-allocation=none" contradicting "--split=N"?

### ISSUE THREAD

**Issue by sdfww** [aria2/aria2#1396]
If -split=N" (N>1) is turned on, "--file-allocation=none" will be invalid.

**Comment by VaslD** [comment]
No... (I’m not exactly sure what you mean so I’m explaining based on what I think you meant.)

`split` mostly decides how many pieces to simultaneously download (i.e. how many simultaneous connections to use for 1 download task), while `file-allocation` decides what to do before actually writing data on the disk (i.e. where to write data on the disk). They don’t have nothing to do with each other.

`split` doesn’t actually split files on the disk, in a general sense. It only controls things on the network side.

**Comment by xinxin8816** [comment]
yep, u are right
none file allocation cant work with split>1
